### PR TITLE
Issue 5143 - Clear console warnings

### LINKED ISSUE #5143: Clear console warnings
### Description
There are many warnings printed as errors that hog the console and add a lot of useless text when trying to read actual errors

### Suggested solution
Fix the parts of the code that log them

### Goal
- [ ] Make console cleaner to read and easier to spot real errors when they happen

### Tasks
- [ ] Identify the warnings and the parts of the code triggering them and fix them

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #5143 

#### Description
I got rid of all the warnings printed as errors that I have bumped into. I might be missing still some though, so feel free to raise list them in case

#### Test cases
Navigate around the app and play with the different components. No warnings should be printed (as error)

### CODE REVIEW

File: frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/commentsPanel.component.tsx [hunk 1]
```diff
@@ -131,6 +131,13 @@ export const CommentsPanel = ({ scrollPanelIntoView }: CommentsPanelProps) => {
 						initialTopMostItemIndex={commentsLength - 1}
 						followOutput={() => true}
 						overscan={800}
+						components={{
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/commentsPanel.component.tsx:134]: what happened here?

File: frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/viewerInputContainer/viewerInputContainer.component.tsx [hunk 2]
```diff
@@ -25,13 +25,13 @@ import { VIEWER_ROUTE, ViewerParams } from '@/v5/ui/routes/routes.constants';
...
 
-export const ViewerInputContainer = (props) => {
+export const ViewerInputContainer = ({ inputRef = undefined, ...props }) => {
 	const { isViewer, containerOrFederation } = useContext(TicketContext);
 	const { teamspace, project } = useParams<ViewerParams>();
 	
 	const ticketId = TicketsCardHooksSelectors.selectSelectedTicketId();
 
-	if (isViewer) return (<InputContainer {...props} />);
+	if (isViewer) return (<InputContainer ref={inputRef} {...props} />);
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/viewerInputContainer/viewerInputContainer.component.tsx:34]: what happened here? there was something not working?

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

**Review by The-Daniel — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/viewerInputContainer/viewerInputContainer.component.tsx (hunk 2 above)
Comment (Amantini1997) [reply frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/viewerInputContainer/viewerInputContainer.component.tsx:34]: Some components were complaining about the `ref` passed to the child input not being populated

File: frontend/assets/icons/filled/logo_text-filled.svg.tsx
[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/assets/icons/filled/logo_text-filled.svg.tsx:1]: The second one is an actual error and, unfortunately, is triggered by MUI, so it won't be fixed

[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/assets/icons/filled/logo_text-filled.svg.tsx:1]: I can't reproduce this one `Errors in invitationDialog (When adding a user to a teamspace and you click the job select).`
Can you record the steps?

**Review by The-Daniel — APPROVED** [review]

**Review by sanmont3drepo — DISMISSED** [review]

**Review by sanmont3drepo — APPROVED** [review]

**Review by The-Daniel — APPROVED** [review]

**Pull request merged**